charts/celery-exporter/README.md

```diff
@@ -11,7 +11,7 @@ Prometheus exporter for Celery
 Add the helm repository:
 
 ```bash
-helm repo add danihodovic https://danihodovic.github.io/helm-charts
+helm repo add danihodovic https://danihodovic.github.io/celery-exporter/
```

Comment:
Or... is this correct?

Reply:
should be - see https://danihodovic.github.io/celery-exporter/index.yaml
Adding the repo from that URL and installing the chart worked for me. 

Existing URL returns

`❯ helm repo add danihodovic https://danihodovic.github.io/celery-exporters
Error: looks like "https://danihodovic.github.io/celery-exporters" is not a valid chart repository or cannot be reached: failed to fetch https://danihodovic.github.io/celery-exporters/index.yaml : 404 Not Found
`

Reply:
https://danihodovic.github.io/celery-exporter seems to work as well.

I don't mind changing it since both work.

Reply:
oh, you meant the trailing slash? Missed it, that is unnecessary